src/profile-logic/function-info.js

```diff
@@ -38,7 +38,11 @@ export function removeTemplateInformation(functionName: string) {
   let result = '';
   let depth = 0;
   let start = 0; // Start of a segment we'd like to keep
-  for (let i = 0; i < functionName.length; i++) {
+  // We start the loop at i = 1 because we don't want to extract any
+  // template-like information starting at the beginning of the string:
+  // templates can't occur before the name of a function, so this is certainly
+  // an HTML tag like <script>.
```

Comment:
I presume we can't have anything else before the actual HTML tag in the string, say, a space, or `"(root scope) "`? In that case we'd still lose the tag.

Reply:
This is a question I asked myself too, and the truth is I don't know :)
Maybe I can make it better by looking at the previous character, to make sure it's not a space...